README.md

```diff
@@ -170,6 +171,32 @@ one character at the time. `false` is the default value.
 are typed. By default it's 0. You can use this option if your component has a
 different behavior for fast or slow users.
 
+#### Special characters
+
+The following special character strings are supported:
+
+| Text string   | Key       | Modifier   | Notes                                                                              |
+| ------------- | --------- | ---------- | ---------------------------------------------------------------------------------- |
+| `{enter}`     | Enter     | N/A        | Will insert a newline character (`<textarea />` only).                             |
+| `{esc}`       | Escape    | N/A        |                                                                                    |
+| `{backspace}` | Backspace | N/A        | Will delete the previous character (or the characters within the `selectedRange`). |
+| `{shift}`     | Shift     | `shiftKey` | Does **not** capitalize following characters.                                      |
+| `{ctrl}`      | Control   | `ctrlKey`  |                                                                                    |
+| `{alt}`       | Alt       | `altKey`   |                                                                                    |
+| `{meta}`      | OS        | `metaKey`  |                                                                                    |
+
+> **A note about modifiers:** Modifier keys (`{shift}`, `{ctrl}`, `{alt}`,
+> `{meta}`) will activate their corresponding event modifiers for the duration
+> of type command or until they are closed (via `{/shift}`, `{/ctrl}`, etc.).
```

Comment:
I do like the extra control we get here by adding `{/shift}` (compared to cypress), but worry that maybe it's a bit too verbose for most use cases.

Perhaps you could release modifier keys at the end of the type statement if no closing modifiers are provided? That could be the best of both worlds. I don't think there's any use case for keeping these keys held down at the end of a type statement (since you don't get the event modifiers after it's done).